test/unit/Schema.js

```diff
@@ -1156,6 +1155,57 @@ describe("Schema", () => {
 		});
 	});
 
+	describe("getIndexAttributes", () => {
+		const tests = [
+			// Defaults
```

Comment:
I don't think this matches what we are doing here.

Reply:
Another copy/paste artifact. 

Resolved in https://github.com/dynamoose/dynamoose/pull/1079/commits/ca9f42593e058b16b7ba2246b91ff42acf6bc16e